### ISSUE TITLE
Crashed ViaFabric Plus 

### ISSUE THREAD

**Issue by Sofutikku** [ViaVersion/ViaFabricPlus#543]
### General Information

https://mclo.gs/FaG1WRE#L77 - crash report 

### Bug Description

Joining the server on 1.20.4 with the latest version ViaFabric Plus 1.21 and the game crashes

### Steps to Reproduce

1. I indicate version 1.20.3-1.20.4
2. Join the server with mc.cubach.com
2. Crash

### Expected Behavior

The game should not crash

**Comment by Yawarakaii** [comment]
> Provide the full client logs please

[latest.log](https://github.com/user-attachments/files/16741382/latest.log)

**Comment by SpigotRCE** [comment]
does this still persists?

**Comment by 2No2Name** [comment]
Yes, according to the two above reports being marked as duplicate

**Comment by 2No2Name** [comment]
What do you need help with?

**Comment by FlorianMichael** [comment]
@2No2Name I can reproduce this issue even without Lithium, so it's not caused by an incompatibility.

I looked into it previously, and it seemed a bit more complex due to how we handle chunks. I marked it as help wanted in case someone wants to do me a favor with debugging.

**Comment by HowardZHY** [comment]
Is it still valid?
